src/Atmos/Model/turbulence.jl

```diff
@@ -108,24 +107,31 @@ eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.2153-3490.1962.tb001
 year = {1962}
 }
 """
-function buoyancy_correction(S, diffusive::Vars, aux::Vars)
+function squared_buoyancy_correction(normS, diffusive::Vars, aux::Vars)
   T = eltype(diffusive)
   N² = inv(aux.moisture.θ_v * diffusive.turbulence.∂θ∂Φ)
-  normS = sqrt(2*(S[1]^2 + S[2]^2 + S[3]^2 + 2*(S[4]^2 + S[5]^2 + S[6]^2)))
   Richardson = N² / (normS^2 + eps(normS))
-  buoyancy_factor = N² <= T(0) ? T(1) : sqrt(max(T(0), T(1) - Richardson*inv_Pr_turb))^(T(1//4))
-  return buoyancy_factor
+  if N² <= T(0)
+    T(1) 
+  else  
+    max(T(0), sqrt(T(1) - Richardson*inv_Pr_turb))
```

Comment:
`sqrt(max(T(0), T(1) - Richardson * inv_Pr_turb))`
(`Richardson` can be large if shear contribution is low)